docs/Configuration.md

```diff
@@ -13,6 +13,7 @@ Below are the different initialization options available for the Agent:
 |`allowInsecureBeacon`|`boolean`|`false`|Allows sending data to insecure endpoints not using `https`. It is not recommended to enable this. |
 |`debug`|`boolean`|`false`|Enables debug logging in developer console|
 |`ignoreUrls`|`(string\|regex)[]`|`[]`|Sets a list of URLs to be ignored. Any URLs that XHR/Fetch or websocket connect that match the rules will not produce a span. Two different rules can be provided: equals match to a specified string or a regex match.|
+|`cookieDomain`|`string`|`(none)`|Configure the domain [used for session tracking](Cookies.md). For example if you have sites `foo.example.com` and `bar.example.com`, setting this to `example.com` allows both sites to use same session identifier.|
 |`context.async`|`boolean`|`false`|Enable [asyncronous context manager](Async-Traces.md)|
 |`exporter.onAttributesSerializing`|`(a: SpanAttributes, s?: Span) => SpanAttributes`|`(attrs) => attrs`|Provides a callback for modifying span attributes before exporting. An example use case of the serializer is to filter out PII.|
```

Comment:
```suggestion
|`cookieDomain`|`string`|`(none)`|Configures the domain used for [session tracking](Cookies.md). For example, if you have sites `foo.example.com` and `bar.example.com`, setting `cookieDomain` to `example.com` allows both sites to use the same session identifier.|
|`context.async`|`boolean`|`false`|Enables [asyncronous context manager](Async-Traces.md)|
```

Reply:
accidental `exporter.onAttributesSerializing` line removal in suggestion?